charts/hush-sensor/values.yaml

```diff
@@ -110,6 +110,12 @@ image:
   vermonRepository: vermon
 
 daemonSet:
+  # hush-sensor daemon set
+  #   true  - Deploy hush-sensor DaemonSet as part of helm-chart installation
+  #   false - Do not deploy hush-sensor DaemonSet.
+  #           Useful when specific functionality subset is required.
+  enabled: true
```

Comment:
i know it might sound ugly, but if we define this one as negative:
`disabled: false`

would this make the helm back compat?

Reply:
Probably yes, but this is very unusual.

Reply:
would like to reduce friction.
we have the risk of someone doing upgrade without reset-and-reuse

@horomena what are your thoughts? i prefer to be back compat.

Reply:
Ack, found something that can be done while keeping with positive logic.